src/slices/room.js

```diff
@@ -57,6 +58,9 @@ const roomSlice = createSlice({
     toggleAudioRanges(state) {
       state.audioRangesEnabled = !state.audioRangesEnabled;
     },
+    toggleLightMode(state) {
+      state.lightMode = !state.lightMode;
+    },
```

Comment:
Just writing some redux, nbd 😎 Nice! 🎉 

Under normal non-Hackathon conditions, we would likely create a new slice for settings where we'd store app-wide data/reducers like this. Def no need to change anything here though! Just noting